package.json

```diff
@@ -231,6 +233,7 @@
     "yarn-deduplicate": "^3.1.0"
   },
   "dependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
please move this package to `devDependencies`

Reply:
Ah! Missed this. Done.

Reply:
Can you trigger the build please? To test the E2E suite.